src/css/community.less

```diff
@@ -135,3 +135,14 @@ body.community {
     }
   }
 }
+
+.login-signup-modal .modal-content {
```

Comment:
looks like this was copied from project.less -- can you just remove it from there and here and add it to the bottom of entrance.less?